crates/mun_codegen/src/ir/types.rs

```diff
@@ -1,4 +1,5 @@
 use crate::value::{AsValue, IrValueContext, SizedValueType, TransparentValue, Value};
+use itertools::Itertools;
```

Comment:
This doesn't seem to be used.

Reply:
It is required for the `AsValue` macro. I tried including it in the macro itself, but that didn't work. If you have any alternative ideas, please let me know.

Reply:
I believe you can also put the use in the generated itself, in the function.

Reply:
I'll try it again, but last time I did that it didn't work 😞